src/main/java/core/basesyntax/service/impl/CsvHandlerImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileHandler;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvHandlerImpl implements FileHandler {
+    @Override
+    public List<String> readFile(Path filePath) {
+        List<String> inputData;
+        try {
+            inputData = Files.readAllLines(filePath, Charset.defaultCharset());
+        } catch (IOException e) {
+            throw new RuntimeException("Something wrong went with the file: "
+                    + filePath);
+        }
+        return inputData;
+    }
+
+    @Override
+    public void writeCsv(List<String> report, Path toPath) {
+        try {
+            Files.write(toPath, report);
+        } catch (IOException e) {
+            throw new RuntimeException("Something went wrong during writing"
+                    + " process to file: " + toPath);
+        }
+    }
```

Comment:
Maybe it will be better to do these different things in different classes?
I remember some things if you can explain interface or class like, it makes something and something else, it is bad.
So if you want only to write file you need to use a class that can only write file, not both write and read.